MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -0,0 +1,206 @@
+#if defined(__IPHONE_10_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
```

Comment:
I don't think that you need this, as soon as you weak link the UserNotifications framework you should be good.